roles/lib/files/FWO.Mail/EmailConnection.cs

```diff
@@ -6,18 +6,18 @@ public class EmailConnection
     {
         public string ServerAddress { get; set; } = "";
         public int Port { get; set; }
-        public bool Tls { get; set; }
+        public string Encryption { get; set; } = "plain"; // possible values: plain, tls, starttls
```

Comment:
I would define an enum to avoid "magic strings" here and in the other sources

Reply:
done